docs/ssvc/SSVC.md

```diff
@@ -1,27 +1,35 @@
 # Stakeholder-Specific Vulnerability Categorization (SSVC)
 
+<!-- 
+note: the SSVC documentation site certcc.github.io/SSVC is slated to go online in mid-March 2024.
+Until then, it just redirects to an SSVC calculator https://certcc.github.io/SSVC/ssvc-calc/
+-->
+
 !!! abstract "Overview"
 
-    This section covers [Stakeholder-Specific Vulnerability Categorization](https://resources.sei.cmu.edu/library/asset-view.cfm?assetid=653459)
+    This section covers [Stakeholder-Specific Vulnerability Categorization](https://certcc.github.io/SSVC/)
```

Comment:
Assuming our timing works out, the github.io link will be the authoritative SSVC documentation once it launches.